plugins/inputs/win_perf_counters/win_perf_counters.go

```diff
@@ -349,12 +363,20 @@ func (m *Win_PerfCounters) Gather(acc telegraf.Accumulator) error {
 		if len(instance.instance) > 0 {
 			tags["instance"] = instance.instance
 		}
-		acc.AddFields(instance.name, fields, tags)
+		acc.AddFields(instance.name, fields, tags, timestamp)
 	}
 
 	return nil
 }
 
+//returns true if err is an error we count with
+func isKnowError(err error) bool {
```

Comment:
Can you rename this function and redo the comment?  I think based on your comment above maybe you could name it `isCounterDataError`.  You can probably remove the comment above if this name is more precise.